src/code42cli/cmds/securitydata.py

```diff
@@ -143,6 +145,89 @@
     cls=incompatible_with(["advanced_query", "type", "saved_search"]),
     help="Get all events including non-exposure events.",
 )
+risk_indicator_map = {
+    "PUBLIC_CORPORATE_BOX": RiskIndicator.CloudDataExposures.PUBLIC_CORPORATE_BOX,
```

Comment:
It looks like the docs currently display the original options ("Public link from corporate Box" instead of "PUBLIC_CORPORATE_BOX").  I haven't done much work with the docs in general yet, but I'll look at figuring out how to make them match the help statement (printing the list of map values instead of full strings)

Reply:
Side note - the sphinx-click docs state that the `:show-nested:` flag is being deprecated and replaced by just `:nested:`, would it be good to switch those out at some point?

Reply:
It wouldn't hurt to change those if you're able to!